### DISCUSSION TITLE
Quarkus 3.9.2 released - Maintenance release

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#39972] [Quarkus Blog/Website]
# Quarkus 3.9.2 released - Maintenance release

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-9-2-released/

**Comment by bniculea** [comment]
Hey, do you know where I can find with what GraalVM version is this 3.9.2 version compatible? I am asking because I am trying to create a native build and it complains a lot about some Netty classes.
I have GraalVM  Oracle GraalVM 17.0.10+11.1

**Reply by gsmet** [reply]
You should preferably use GraalVM for Java 21 (and this even if your application is compatible with Java 17).

That being said, we fixed a few issues related to the Netty update for the upcoming 3.9.3 that should be released tomorrow.

**Reply by bniculea** [reply]
Thanks a lot! I was already installing the 21 version and it is working! Really appreciate you fast answer! All the best!

**Reply by gsmet** [reply]
Happy to help!
